00_hello_world/main.go

```diff
@@ -9,5 +9,18 @@ import (
 var out io.Writer = os.Stdout
 
 func main() {
-	fmt.Fprintln(out, "Hallo du schöne Welt!")
+
+	fibonacciSeries(7)
+}
+
+func fibonacciSeries(number int) {
+	var a, b = 0, 1
```

Comment:
Can be simplified to:
```go
	a, b := 1, 1
	for i := 1; i <= n; i++ {
		fmt.Fprintln(out, a)
		a, b = b, a+b
	}
```
Please remove unnecessary blank line at beginning/end of block

Reply:
Done